.github/workflows/build-and-upload.yml

```diff
@@ -71,7 +76,17 @@ jobs:
             export IMAGE_TAG=$KEYCLOAK_REPOSITORY:$(echo ${GITHUB_REF_NAME} | awk '{print tolower($0)}' | sed -e 's|/|-|')
           fi
 
-          cd quarkus/container
-          docker build --build-arg KEYCLOAK_DIST=$(ls keycloak-*.tar.gz) . -t ${IMAGE_TAG}
+          echo "image-url-with-tag=${IMAGE_TAG}" >> $GITHUB_OUTPUT
+          echo "keycloak-archive-name=$(ls keycloak-*.tar.gz)" >> $GITHUB_OUTPUT
```

Comment:
Is this working as intended? Logs for this step say
```
ls: cannot access 'keycloak-*.tar.gz': No such file or directory
```

Reply:
You are right. I've updated the Dockerfile and pipeline and make it explicit to our archive. The default Dockerfile from Keycloak uses the archive from GitHub and there is an option to use local archives. In our case we don't want to support external URLs and there should be an error if the archive is not there which wasn't the case before.
Please see my commits.

Reply:
Awesome, thanks!